core/src/luajit/makefile

```diff
@@ -1,5 +1,7 @@
 # prefix
 include $(PRO_DIR)/prefix.mak
+# architecture mappings
+include $(PRO_DIR)/arch.mak
 
```

Comment:
We need not include arch.mak file here, please move to core/makefile

https://github.com/xmake-io/xmake/blob/f4781b3cbc4d8e61eddb888ea4e7d258adc70c5b/core/makefile#L141-L176